src/components/interactive-builder/modals/question/question-form/rendering-types/inputs/workspace-launcher/workspace-launcher.test.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import WorkspaceLauncher from './workspace-launcher.component';
+import { FormFieldProvider } from '../../../../form-field-context';
+import type { FormField } from '@openmrs/esm-form-engine-lib';
+
+const mockSetFormField = jest.fn();
+const formField: FormField = {
+  type: 'obs',
+  questionOptions: {
+    rendering: 'workspace-launcher' as const,
+    buttonLabel: '',
+    workspaceName: '',
+  },
+  id: '1',
+};
+
+jest.mock('../../../../form-field-context', () => ({
+  ...jest.requireActual('../../../../form-field-context'),
+  useFormField: () => ({ formField, setFormField: mockSetFormField }),
+}));
+
+describe('WorkspaceLauncher', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
```

Comment:
Not required since this is done by default - https://github.com/openmrs/openmrs-esm-form-builder/blob/d80c22868c17a39df4838563947eeed81c9f9ffb/jest.config.js#L3
```suggestion
  beforeEach(() => {
    jest.clearAllMocks();
  });
```